### PR TITLE
Add FAQ section

### PR DESCRIPTION
**Pull request by AndreiKingsley**

### CODE REVIEW

File: docs/topics/Welcome.topic [hunk 1]
```diff
@@ -31,6 +31,7 @@
             <a href="Quick-Start-Guide.md" type="check-list">Quick start guide</a>
             <a href="Kandy-in-Kotlin-Notebook.md" type="computer"/>
             <a href="Statistics-Guide.md" type="network"/>
+            <a href="FAQs.md" type="search"/>
```
Comment (devcrocod) [inline docs/topics/Welcome.topic:34]: "search" doesn't suit for FAQ section
mb use "community" or "support" icon

File: docs/topics/FAQs.md [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+</web-summary>
+<card-summary>
+    Find answers to the most frequently asked questions about Kandy
+</card-summary>
+<link-summary>
+    Answers to the most frequently asked questions about Kandy
+</link-summary>
+
+
+## Where can I ask a question about Kandy, leave feedback or request a feature?
```
Comment (devcrocod) [inline docs/topics/FAQs.md:14]: This can be without a question, just as an introductory paragraph

**Review by devcrocod — CHANGES_REQUESTED** [review]
Need to rewrite all answers

**Comment by AndreiKingsley** [comment]
@devcrocod please review FAQ doc again. If it's OK, I send it for proofreading.

**Comment by devcrocod** [comment]
@AndreiKingsley 
I don't like the last question because it's quite narrow. And the answer turns out to be the same
Otherwise 👍

**Review by devcrocod — APPROVED** [review]

File: docs/topics/FAQ.md [hunk 3]
```diff
@@ -0,0 +1,87 @@
...
+</web-summary>
+<card-summary>
+    Find answers to the most frequently asked questions about Kandy
+</card-summary>
+<link-summary>
+    Answers to the most frequently asked questions about Kandy
+</link-summary>
+
+
+We’re keen to receive any Kandy-related feedback, questions, feature requests, or suggestions you may have.
```
Comment (zaleslaw) [inline docs/topics/FAQ.md:14]: ' instead of  ,

File: docs/topics/FAQ.md [hunk 4]
```diff
@@ -0,0 +1,87 @@
...
+[Kandy GitHub repository](https://github.com/Kotlin/kandy). 
+You can also ask your questions or start a discussion on anything related to Kandy 
+on the [#datascience](https://slack-chats.kotlinlang.org/c/datascience?_ga=2.2771440.1927802643.1705492406-2056581211.1698239845&_gl=1*i9jk3b*_ga*MjA1NjU4MTIxMS4xNjk4MjM5ODQ1*_ga_9J976DJZ68*MTcwNTU3MjM0NC42NC4xLjE3MDU1NzIzNjUuMzkuMC4w) 
+Kotlin Slack channel.
+
+
+## Does kandy support Compose Multiplatform?
+ * We’re looking into it, but currently Kandy’s only target is JVM. 
+If you need Compose Multiplatform support, please follow the
+[corresponding issue](https://github.com/Kotlin/kandy/issues/27) for updates and cast your vote there as well.
```
Comment (zaleslaw) [inline docs/topics/FAQ.md:25]: remove `as well`

File: docs/topics/FAQ.md [hunk 5]
```diff
@@ -0,0 +1,87 @@
...
+([Let's Plot Java Swing Frontend / JavaFX Frontend](https://github.com/JetBrains/lets-plot-kotlin/blob/master/USAGE_BATIK_JFX_JS.md), 
+or [Let’s-Plot Skia Frontend)](https://github.com/JetBrains/lets-plot-skia))
+See an [example app](https://github.com/AndreiKingsley/kandy-swing-app-example).
+ * Please follow the [corresponding issue](https://github.com/Kotlin/kandy/issues/283) for updates 
+and cast your vote there as well.
+ * You can export Kandy Plot to `BufferedImage`, SVG, PNG, JPG, or HTML. 
+See an [example](https://kotlin.github.io/kandy/quick-start-guide.html#export).
+
+
+## Is there geo tools/geo visualization support?
```
Comment (zaleslaw) [inline docs/topics/FAQ.md:47]: `are ` instead of `is`

File: docs/topics/FAQ.md [hunk 6]
```diff
@@ -0,0 +1,87 @@
...
+ * At the moment, no. We’re currently looking into adding support for 
+[GeoTools](https://www.geotools.org/) using 
+[lets-plot-geo](https://github.com/JetBrains/lets-plot-kotlin/blob/master/docs/geotools.md). 
+If you need this support, please follow the [corresponding issue](https://github.com/Kotlin/kandy/issues/280) 
+for updates and cast your vote there as well. 
+It would also be appreciated if you could describe your case and current toolset.
```
Comment (zaleslaw) [inline docs/topics/FAQ.md:53]: Please describe your case and current toolset

File: docs/topics/FAQ.md [hunk 7]
```diff
@@ -0,0 +1,87 @@
...
+
+
+## Does Kandy support 3D plotting?
+ * At the moment, no. This is a limitation of the Lets-Plot engine, 
+which specializes in 2D charts only. If you need this support, 
+please follow the [corresponding issue](https://github.com/Kotlin/kandy/issues/209) 
+for updates and cast your vote there as well.
+
+## Does Kandy support dynamic plotting / animations?
```
Comment (zaleslaw) [inline docs/topics/FAQ.md:62]: plotting/animations instead of plotting / animations

File: docs/topics/FAQ.md [hunk 8]
```diff
@@ -0,0 +1,87 @@
...
+ * Kandy supports tooltips: See the [documentation](https://kotlin.github.io/kandy/tooltips-guide.html).
+ * At the moment, the Lets-Plot version of Kandy doesn’t support any kind of animations
```
Comment (zaleslaw) [inline docs/topics/FAQ.md:64]: At the moment, the Lets-Plot version of Kandy doesn't support any animations

File: docs/topics/FAQ.md [hunk 9]
```diff
@@ -0,0 +1,87 @@
...
+or dynamic data plotting. This is a limitation of the Lets-Plot library,
+which specializes in static charts. If you need this support,
+please create an issue describing your use case 
+ * The Apache Echarts (a JavaScript plotting library) engine for Kandy 
+(experimental; still in development) supports different kinds of animations and can work with dynamic data.
+
+
+## My use case involves creating several layers with multiple series of data, and I can’t figure out how to create a legend for them (for their colors, for example). Is it possible?
```
Comment (zaleslaw) [inline docs/topics/FAQ.md:72]: with multiple data series

Comment (zaleslaw) [reply docs/topics/FAQ.md:72]: My use case involves creating several layers with multiple data series, and I need help figuring out how to make a legend for them (for their colors, for example). Is it possible?

File: docs/topics/FAQ.md [hunk 10]
```diff
@@ -0,0 +1,87 @@
...
+parameter without mapping on this parameter. 
+That means you can’t create several layers and distinguish them using legend. 
+We’re currently working on support for this approach (which we call a “series” approach).
+For the time being, you can use a [series hack](https://kotlin.github.io/kandy/series-hack-guide.html) 
+solution as a workaround. 
+ * The Apache Echarts engine for Kandy (experimental; still in development) supports this functionality.
+
+## My use case involves creating several bars/areas/other layers that are overlapped. Is there a way to adjust their position?
+ * Lets-Plot does not allow for any customization between different layers, 
+so at the moment the only solution is to use a [series hack](https://kotlin.github.io/kandy/series-hack-guide.html)
```
Comment (zaleslaw) [inline docs/topics/FAQ.md:84]: so, at the moment,

File: docs/topics/FAQ.md [hunk 11]
```diff
@@ -0,0 +1,87 @@
...
+ * At the moment, no. We’re currently looking into adding support for 
+[GeoTools](https://www.geotools.org/) using 
+[lets-plot-geo](https://github.com/JetBrains/lets-plot-kotlin/blob/master/docs/geotools.md). 
+If you need this support, please follow the [corresponding issue](https://github.com/Kotlin/kandy/issues/280) 
+for updates and cast your vote there as well. 
+It would also be appreciated if you could describe your case and current toolset.
+
+
+## Does Kandy support 3D plotting?
+ * At the moment, no. This is a limitation of the Lets-Plot engine,
```
Comment (zaleslaw) [inline docs/topics/FAQ.md:57]: There is?

**Review by zaleslaw — APPROVED** [review]
Structure and questions are cool, I've just added some simplifications in some cases

File: docs/topics/FAQ.md (hunk 3 above)
Comment (AndreiKingsley) [reply docs/topics/FAQ.md:14]: ?

File: docs/topics/FAQ.md (hunk 5 above)
Comment (AndreiKingsley) [reply docs/topics/FAQ.md:47]: Why?

File: docs/topics/FAQ.md (hunk 6 above)
Comment (AndreiKingsley) [reply docs/topics/FAQ.md:53]: Why?

File: docs/topics/FAQ.md (hunk 9 above)
Comment (AndreiKingsley) [reply docs/topics/FAQ.md:72]: 1. Ok
2. Why?

File: docs/topics/FAQ.md (hunk 11 above)
Comment (AndreiKingsley) [reply docs/topics/FAQ.md:57]: Why?

File: docs/topics/FAQ.md (hunk 7 above)
Comment (devcrocod) [reply docs/topics/FAQ.md:62]: It depends on how the chapter looks on the web

File: docs/topics/FAQ.md (hunk 9 above)
Comment (devcrocod) [reply docs/topics/FAQ.md:72]: I disagree
"Data Series" is a common name for many things like in pandas, in some tools from microsoft, google. So "data series" may cause confusion among some people

**Pull request merged**